corehq/apps/export/models/new.py

```diff
@@ -648,13 +648,13 @@ def is_location_safe_for_user(self, request):
 
 class CaseExportInstanceFilters(ExportInstanceFilters):
     sharing_groups = ListProperty(StringProperty)
-    show_all_data = BooleanProperty(default=True)
-    show_project_data = BooleanProperty()
+    show_all_data = BooleanProperty()
+    show_project_data = BooleanProperty(default=True)
     show_deactivated_data = BooleanProperty()
 
 
 class FormExportInstanceFilters(ExportInstanceFilters):
-    user_types = ListProperty(IntegerProperty, default=[HQUserType.ACTIVE])
+    user_types = ListProperty(IntegerProperty, default=[HQUserType.ACTIVE, HQUserType.DEACTIVATED])
```

Comment:
what does this change do?

Reply:
It adds `[Deactivated Mobile Users]` to the default user filters on exports, so now the default filters are `[Active Mobile Users] [Deactivated Mobile Users]`

Reply:
Oh, is this just for form exports? woah, I thought this was already the default, good catch